### DISCUSSION TITLE
Is There a Way to Import Text Into a .BAS File?

### DISCUSSION THREAD

**Discussion by Paladin1870** [robhagemans/pcbasic#248] [General]
I have a PC-BASIC game design in mind which would need 48 similar GOSUB routines. In PC-BASIC I'd have to list one and change the line numbers to make the GOSUB copies (and then minor changes to each line). Doing copy-and-paste in a text editor and importing the lines of code would be much easier. Can this be done? Or does anyone have some other idea? Thanks!

**Comment by Marrin** [comment]
Of course this can be done.  BASIC programs can be loaded and saved as text files.  Loading doesn't need special care.  Saving needs the `,A` parameter.

**Comment by Paladin1870** [comment]
Hi!  Thanks so much!

Andrew
